content/installation/setup/auth/Authentication.md

```diff
@@ -10,6 +10,7 @@ Contrast supports a variety of authentications providers:
 * LDAP
 * Active Directory
 * Single Sign-On
+* HTTP Header
```

Comment:
This was the wording used in the UI. Will have to double check with Product. I'll get back to you on this @kaitlynmortimer